pyanaconda/core/constants.py

```diff
@@ -66,6 +66,11 @@
                  "rawhide",
                  "BaseOS"]
 
+# Get list of repo names which should be used as updates repos
+DEFAULT_UPDATE_REPOS = [productName.split('-')[0].lower() + '-updates',
```

Comment:
Please, remove the first repo from the list. We enable or disable only `updates` and `updates-modular` at this moment.

Reply:
Right now `DEFAULT_REPOS` can find a repo off of productName for the BaseOS.  Adding similar behavior for a productName specific updates repo would make them more parallel.

Reply:
Yes, you are right but we would like to change this solution completely. We would like to avoid matching stuff based on product name it is not that clear how this should work and it breaks custom images often. The only thing using this is Fedora now so there is no reason to have this shadow logic just for one system. Also you are blocking systems based on Fedora in renaming their repos / product names.

Or do you want to use this somewhere else?

Reply:
My use case is fairly soft and I can solve another way.  I'll drop the productName updates and update the patch.